src/middleware/auth.middleware.js

```diff
@@ -0,0 +1,22 @@
+import { jwtService } from '../services/jwt.service.js';
+
+export const authMiddleware = (req, res, next) => {
```

Comment:
The middleware function should ideally set the user data in the request object if the token is valid. This allows subsequent middleware or route handlers to access the authenticated user's information.